scripts/commands/updateApiDocs.ts

```diff
@@ -47,7 +47,7 @@ interface Arguments {
   package: string;
   version: string;
   historical: boolean;
-  artifact: string;
+  "skip-download": boolean;
```

Comment:
this can be skipDownload, e.g.

https://github.com/Qiskit/documentation/blob/7439931636ba19620200508ea9283a0ec7888d62/scripts/commands/checkLinks.ts#L32